corehq/apps/es/case_search.py

```diff
@@ -216,6 +238,18 @@ def case_property_range_query(case_property_name, gt=None, gte=None, lt=None, lt
     )
 
 
+def case_ids_lookup(domain, property_name, value, op='='):
```

Comment:
nit: this function might belong more in a `dbaccessors` or `esaccessors` file, since it's a standalone accessor, rather than a component of the CaseSearchES query building library.

Reply:
I'm also not sure how to do that beyond naming functions so they reflect they deal with related cases, which does rely on the reader to know that since ES doesn't join, "related cases" means another query.